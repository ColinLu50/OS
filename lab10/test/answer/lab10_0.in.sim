6359
185
187
172
185
171
