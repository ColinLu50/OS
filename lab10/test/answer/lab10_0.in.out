FCFS
14 -> 164 -> 148 -> 16 -> 38 -> 46 -> 112 -> 3 -> 122 -> 66 -> 30 -> 24 -> 146 -> 126 -> 142 -> 151 -> 65 -> 123 -> 107 -> 64 -> 17 -> 135 -> 21 -> 66 -> 83 -> 132 -> 48 -> 85 -> 129 -> 122 -> 1 -> 173 -> 107 -> 104 -> 92 -> 41 -> 102 -> 51 -> 85 -> 170 -> 117 -> 9 -> 10 -> 88 -> 87 -> 113 -> 76 -> 87 -> 143 -> 17 -> 161 -> 77 -> 142 -> 121 -> 69 -> 173 -> 52 -> 124 -> 135 -> 156 -> 133 -> 50 -> 118 -> 57 -> 46 -> 92 -> 56 -> 3 -> 150 -> 106 -> 153 -> 134 -> 103 -> 151 -> 73 -> 154 -> 80 -> 73 -> 42 -> 6 -> 23 -> 129 -> 4 -> 169 -> 110 -> 8 -> 157 -> 63 -> 10 -> 12 -> 10 -> 39 -> 103 -> 135 -> 13 -> 161 -> 33 -> 15 -> 157 -> 53 -> 52 -> 39 -> 63 -> 62 -> 134 -> 71
6359
SSTF
14 -> 13 -> 12 -> 10 -> 10 -> 10 -> 9 -> 8 -> 6 -> 4 -> 3 -> 3 -> 1 -> 15 -> 16 -> 17 -> 17 -> 21 -> 23 -> 24 -> 30 -> 33 -> 38 -> 39 -> 39 -> 41 -> 42 -> 46 -> 46 -> 48 -> 50 -> 51 -> 52 -> 52 -> 53 -> 56 -> 57 -> 62 -> 63 -> 63 -> 64 -> 65 -> 66 -> 66 -> 69 -> 71 -> 73 -> 73 -> 76 -> 77 -> 80 -> 83 -> 85 -> 85 -> 87 -> 87 -> 88 -> 92 -> 92 -> 102 -> 103 -> 103 -> 104 -> 106 -> 107 -> 107 -> 110 -> 112 -> 113 -> 117 -> 118 -> 121 -> 122 -> 122 -> 123 -> 124 -> 126 -> 129 -> 129 -> 132 -> 133 -> 134 -> 134 -> 135 -> 135 -> 135 -> 142 -> 142 -> 143 -> 146 -> 148 -> 150 -> 151 -> 151 -> 153 -> 154 -> 156 -> 157 -> 157 -> 161 -> 161 -> 164 -> 169 -> 170 -> 173 -> 173
185
SCAN
14 -> 13 -> 12 -> 10 -> 9 -> 8 -> 6 -> 4 -> 3 -> 1 -> 0 -> 15 -> 16 -> 17 -> 21 -> 23 -> 24 -> 30 -> 33 -> 38 -> 39 -> 41 -> 42 -> 46 -> 48 -> 50 -> 51 -> 52 -> 53 -> 56 -> 57 -> 62 -> 63 -> 64 -> 65 -> 66 -> 69 -> 71 -> 73 -> 76 -> 77 -> 80 -> 83 -> 85 -> 87 -> 88 -> 92 -> 102 -> 103 -> 104 -> 106 -> 107 -> 110 -> 112 -> 113 -> 117 -> 118 -> 121 -> 122 -> 123 -> 124 -> 126 -> 129 -> 132 -> 133 -> 134 -> 135 -> 142 -> 143 -> 146 -> 148 -> 150 -> 151 -> 153 -> 154 -> 156 -> 157 -> 161 -> 164 -> 169 -> 170 -> 173
187
C-SCAN
14 -> 13 -> 12 -> 10 -> 9 -> 8 -> 6 -> 4 -> 3 -> 1 -> 0 -> 173 -> 173 -> 170 -> 169 -> 164 -> 161 -> 157 -> 156 -> 154 -> 153 -> 151 -> 150 -> 148 -> 146 -> 143 -> 142 -> 135 -> 134 -> 133 -> 132 -> 129 -> 126 -> 124 -> 123 -> 122 -> 121 -> 118 -> 117 -> 113 -> 112 -> 110 -> 107 -> 106 -> 104 -> 103 -> 102 -> 92 -> 88 -> 87 -> 85 -> 83 -> 80 -> 77 -> 76 -> 73 -> 71 -> 69 -> 66 -> 65 -> 64 -> 63 -> 62 -> 57 -> 56 -> 53 -> 52 -> 51 -> 50 -> 48 -> 46 -> 42 -> 41 -> 39 -> 38 -> 33 -> 30 -> 24 -> 23 -> 21 -> 17 -> 16 -> 15
172
LOOK
14 -> 13 -> 12 -> 10 -> 9 -> 8 -> 6 -> 4 -> 3 -> 1 -> 15 -> 16 -> 17 -> 21 -> 23 -> 24 -> 30 -> 33 -> 38 -> 39 -> 41 -> 42 -> 46 -> 48 -> 50 -> 51 -> 52 -> 53 -> 56 -> 57 -> 62 -> 63 -> 64 -> 65 -> 66 -> 69 -> 71 -> 73 -> 76 -> 77 -> 80 -> 83 -> 85 -> 87 -> 88 -> 92 -> 102 -> 103 -> 104 -> 106 -> 107 -> 110 -> 112 -> 113 -> 117 -> 118 -> 121 -> 122 -> 123 -> 124 -> 126 -> 129 -> 132 -> 133 -> 134 -> 135 -> 142 -> 143 -> 146 -> 148 -> 150 -> 151 -> 153 -> 154 -> 156 -> 157 -> 161 -> 164 -> 169 -> 170 -> 173
185
C-LOOK
14 -> 13 -> 12 -> 10 -> 9 -> 8 -> 6 -> 4 -> 3 -> 1 -> 173 -> 170 -> 169 -> 164 -> 161 -> 157 -> 156 -> 154 -> 153 -> 151 -> 150 -> 148 -> 146 -> 143 -> 142 -> 135 -> 134 -> 133 -> 132 -> 129 -> 126 -> 124 -> 123 -> 122 -> 121 -> 118 -> 117 -> 113 -> 112 -> 110 -> 107 -> 106 -> 104 -> 103 -> 102 -> 92 -> 88 -> 87 -> 85 -> 83 -> 80 -> 77 -> 76 -> 73 -> 71 -> 69 -> 66 -> 65 -> 64 -> 63 -> 62 -> 57 -> 56 -> 53 -> 52 -> 51 -> 50 -> 48 -> 46 -> 42 -> 41 -> 39 -> 38 -> 33 -> 30 -> 24 -> 23 -> 21 -> 17 -> 16 -> 15
171
